Packaging/nix/devilutionx.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.diasurgical.DevilutionX</id>
+  
+  <name>DevilutionX</name>
+  <summary>Diablo/Hellfire source port for modern operating systems</summary>
+  
+  <metadata_license>CC-BY-SA-4.0</metadata_license>
+  <project_license>UPL-1.0</project_license>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+    <control>gamepad</control>
+  </supports>
+  
+  <description>
+    <p>
+      DevilutionX is a source port of Diablo and Hellfire that strives to make it simple to run the game while providing engine improvements, bugfixes, and some optional quality of life features. This includes support for modern operating systems and higher resolutions, controller and touch control support, stash, optional experience and enemy health bars, etc.
+    </p>
```

Comment:
Thinking more about it, I think "reimplementation" is more appropriate for project description.
We're not a "source port" since Diablo's source was never released.

And "build" is not appropriate since the scope of the project went FAR beyond just compatibility with modern OS's.

Reply:
It didn't have to be released if we recreated it 100% ...

Reply:
And that's why "reimplementation" is a good word
Since Activision-Blizzard-King never released the source code, you decompiled and reimplemented it yourself

Remember: this text will be seen in stores like Gnome Software Center and KDE Discover, so semantics are important

Reply:
whatever, I've never even heard about flathub / these other things before xD

Reply:
We could align the text probably the text should align between snap craft, Google play, Github, Xbox store and switch homebrew store

Reply:
It would be better to use "reimplementation" everywhere

Reply:
It is definitely a source port. The source was binary exact and then ported to other systems. It didn't start out as something that kind of matched vanilla enough that it kind of worked with the D1 assets.

I strongly disagree with changing it away from source port

Reply:
@Trihedraf source port needs source code to be released.
GZDoom is a source port, FreeSpace 2 Source Code Project is a source port. They took the code that was released and changed it with new features.

Diablo never had a source code release, Devilution is a decompilation. Decompilation =/= source code release, even if the code is binary exact, and definitely doesn't save us from litigation (see fate of re3, GTA3 decompilation).

If you disagree that strongly, let's move discussion to Discord